src/main/kotlin/com/fasterxml/jackson/module/kotlin/ConstructorValueCreator.kt

```diff
@@ -0,0 +1,12 @@
+package com.fasterxml.jackson.module.kotlin
+
+import kotlin.reflect.KFunction
+import kotlin.reflect.jvm.isAccessible
+
+internal class ConstructorValueCreator<T>(override val callable: KFunction<T>) : ValueCreator<T>() {
+    override val accessible: Boolean = callable.isAccessible
+
+    init {
+        callable.isAccessible = true
+    }
```

Comment:
Does this override the above since this `init` block is lower? Is that the correct behavior?

Reply:
Are you saying that it is difficult to know which comes first, initializing the property or rewriting `isAccessible`?
If I believe `Intellij IDEA`'s point, it should work correctly, but if I value readability, should I rewrite it as follows?(Rewriting the accessible property was surprisingly expensive, so I've included a change to rewrite it only when necessary.)

```kotlin
override val accessible: Boolean = run {
    val initialAccessible = callable.isAccessible

    if (!initialAccessible) callable.isAccessible = true

    initialAccessible
}
```

Reply:
Understood, I was just a bit unsure whether that rewrite was intended 👍